lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -19,61 +19,47 @@
                 )
             )}
         </p>
-    % if not credit_status["purchased"] and not credit_status["error"] :
-        <p class="message-copy credit-eligibility-msg">
-            ${_("You are now eligible for credit. <b>Congratulations!</b>")}
+    <div class="credit-action">
+        % if not credit_status["purchased"] and not credit_status["error"] :
```

Comment:
If this were not a patch, I would have re-written most of this page. Given the lack of automated testing I opted to ignore the obvious code smells and simply update the text.